### DISCUSSION TITLE
Welcome to the Opencast Discussions Test

### DISCUSSION THREAD

**Discussion by gregorydlogan** [opencast/opencast#5319] [Announcements]
## 👋 Welcome!
  Normally our discussions live on our mailing lists at users@opencast.org, but for testing purposes we are opening up GitHub Discussions to talk about:
  * How to make LMS and Tobira work with Static File Authorization

  This is a limited-scope test of Discussions, so for now all other discussion should stay on the mailing lists.  If that changes, you'll see an announcement on list, and this welcome message will change!

**Comment by ziegenberg** [comment]
Finally, GitHub discussions! I very much welcome this test :)

**Reply by lkiesow** [reply]
The test is done. We decided to migrate. See future announcements.
